certbot/CHANGELOG.md

```diff
@@ -10,7 +10,8 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 
 ### Changed
 
-*
+* We deprecated support for Python 2 in Certbot and its ACME library.
+  Support for Python 2 will be removed in the next major release of Certbot.
```

Comment:
nit: We said "major" before, but this phrasing made me think of the major version in semver and I think we're not planning on updating the major version number.

```suggestion
  Support for Python 2 will be removed in the next planned release of Certbot.
```

Reply:
Indeed. By the way, and to make a connection with @atombrella who made a remark about relationship between semver and Python version deprecation: so far we did not consider, like several other major Python projects, that removing the support for an EOL Python version has to be considered as a breaking change requiring a major bump, since it is external to the project.